test_prints_labels_if_given_on_failure is flaky

```
=================================== FAILURES ===================================
____________________ test_prints_labels_if_given_on_failure ____________________
[gw15] linux -- Python 3.6.4 /usr/bin/python
Traceback (most recent call last):
  File "/build/python-hypothesis/src/hypothesis-python-3.44.21/tests/cover/test_arbitrary_data.py", line 67, in test_prints_labels_if_given_on_failure
    assert 'Draw 1 (Some numbers): [0, 0]' in result
AssertionError: assert 'Draw 1 (Some numbers): [0, 0]' in ''
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "/build/python-hypothesis/src/hypothesis-python-3.44.21/tests/common/utils.py", line 52, in raises
    assert False, "Expected to raise an exception but didn't"
AssertionError: Expected to raise an exception but didn't
```

--- a/hypothesis-python/tests/cover/test_arbitrary_data.py
+++ b/hypothesis-python/tests/cover/test_arbitrary_data.py
@@ -52,7 +52,7 @@ def test_prints_on_failure():
 def test_prints_labels_if_given_on_failure():
     @given(st.data())
     def test(data):
-        x = data.draw(st.lists(st.integers(), min_size=1),
+        x = data.draw(st.lists(st.integers(0, 10), min_size=2),
                       label='Some numbers')
         y = data.draw(st.sampled_from(x), label='A number')
         assert y in x

